third_party/move/move-compiler/src/naming/translate.rs

```diff
@@ -12,7 +12,7 @@ use crate::{
     },
     naming::ast as N,
     parser::ast::{Ability_, ConstantName, Field, FunctionName, StructName, Var},
-    shared::{unique_map::UniqueMap, *},
+    shared::{known_attributes::KnownAttribute, unique_map::UniqueMap, CompilationEnv, Name, *},
```

Comment:
Why can this be not contained to expansion? Expansion is also shared with v2, so anything you are doing in naming, would not apply to the v2 compiler chain, and we have to do it again there.

Reply:
I know.  I'll take another look at it.

Reply:
It's now limited to expansion.  Please take a look.